src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+import "./IPauserRegistry.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ * @notice Allows AVSs to make "Range Payments", which get distributed amongst the AVSs' confirmed
+ * Operators and the Stakers delegated to those Operators.
+ * Calculations are performed based on the completed Range Payments, with the results posted in
+ * a Merkle root against which Stakers & Operators can make claims.
+ */
+interface IPaymentCoordinator {
+    /// STRUCTS ///
+    struct StrategyAndMultiplier {
+        IStrategy strategy;
+        // weight used to compare shares in multiple strategies against one another
+        uint96 multiplier;
+    }
+
+    struct RangePayment {
```

Comment:
The term range is a little confusing here.  I'm assuming it's for a time range in the past the Operator/Staker served?  It would be helpful to clarify

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/519
Added comments here